app/services/plan_service/api/api.rb

```diff
@@ -1,7 +1,34 @@
 module PlanService::API
   class Api
+    Configuration = PlanService::DataTypes::Configuration
+
     def self.plans
-      @plans ||= PlanService::API::Plans.new
+      configuration = build_configuration()
```

Comment:
Minor thing probably but now every time the plans is looked up the configuration gets rebuilt unnecessarily when the caching ```@plans ||= ...``` statement comes only after.